BDArmory/Weapons/Missiles/MultiMissileLauncher.cs

```diff
@@ -258,7 +402,27 @@ public void ShipModified(ShipConstruct data)
                                     var explosivePart = missile.FindModuleImplementing<BDExplosivePart>();
                                     tntMass = explosivePart != null ? explosivePart.tntMass : 0;
                                     missileLauncher.blastRadius = BlastPhysicsUtils.CalculateBlastRange(tntMass);
+                                    missileMass = missile.partInfo.partPrefab.mass;
                                     EditorLogic.DeletePart(missile);
+                                    List<Part>.Enumerator sym = part.symmetryCounterparts.GetEnumerator();
```

Comment:
It's better for code maintainability to use `using` instead of explicitly `Dispose`ing the enumerator.